src/main/java/util/HttpRequestUtils.java

```diff
@@ -31,9 +34,13 @@
             return Maps.newHashMap();
         }
 
-        String[] tokens = values.split(separator);
-        return Arrays.stream(tokens).map(t -> getKeyValue(t, "=")).filter(p -> p != null)
-                .collect(Collectors.toMap(p -> p.getKey(), p -> p.getValue()));
+        String decoded = URLDecoder.decode(values, StandardCharsets.UTF_8);
+
+        String[] tokens = decoded.split(separator);
+        return Arrays.stream(tokens)
```

Comment:
변수를 그냥 안에다 넣어도 좋을 것 같군요.

```suggestion
        return Arrays.stream(decoded.split(separator))

Reply:
넵! 수정하겠습니다!